### ISSUE TITLE
Problems by setting the jsonPath?? :)

### ISSUE THREAD

**Issue by jascdk** [goedh452/homebridge-http-sprinkler#2]
Hi

Thanks for a great add-on! I have just got it to work (well almost work).

I have problems about, what I should write in the jsonpath field?

I want to shut on and off a tasmota flashed device 

Best regards

Jacob

**Comment by goedh452** [comment]
Hi Jacob,
What to use as Json path depends on the result you get form the api of your system. In the example config it is based on Domoticz. There the result I use is stored in the path "result[0].level". Result is the opening tag, the 0 is the first record en level is the attribute.

**Comment by jascdk** [comment]
Hi:) Thanks for the answer, but sorry - it did not help me:( I asked on a forum on facebook and got this answer:

Try: stat/[nameOfYouDev]/RESULT
or stat/[nameOfYouDev]/POWER


It did not work either:(


What do you think is should write? I use home assistant, withe the homebridge add-on and my switch is an ESP8266 flashed with tasmota firmware

Best regards

Jacob

**Comment by goedh452** [comment]
Paste the status URL in a browser and paste the result here. Maybe I can see whet the path might be.

**Comment by jascdk** [comment]
This is the result of my status:

 {"Status":{"Module":8,"FriendlyName":["Sonoff Kontor"],"Topic":"kontor","ButtonTopic":"0","Power":1,"PowerOnState":3,"LedState":1,"SaveData":1,"SaveState":1,"ButtonRetain":0,"PowerRetain":1}}

**Comment by goedh452** [comment]
I don’t know which attribute contains the value which indicates if the sprinkler is turned on but let’s say it is Power then the path should be Status.Power or Status[0].Power. The parameter in the config for OnValue should be set to “1” and the offValue to “0”. 

Hope this helps.

**Comment by jascdk** [comment]
Thanks it worked:) But now I have another problem:) When I set the timer to lets say 5 minutes - it begins to count down. That is good. But when I go into the sprinkler control (press the switch in the home app for some seconds) the time keeps resetting back to the original value I have set from start. But it seems like the real time / countdown is happening in the background - it just looks like the app doesn't sync with the right value. I don know if this is the plugins problem or the app itself?
